kiwi_lint/field_label.py

```diff
@@ -0,0 +1,27 @@
+from pylint import checkers
+from pylint import interfaces
+
+
+class FormFieldLabelChecker(checkers.BaseChecker):
```

Comment:
Rename to FormFieldChecker b/c a few more checks can be added here and I'd like to have the name more generic.